api/bootstrap/v1beta1/kzeros_types.go

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Probably this file should be renamed to k0s_types.go as well. I imagine renaming the file won't have any actual impact.
﻿

Reply:
done